### PR TITLE
InfoTooltip reusable component

### PR DESCRIPTION
**Pull request by umrzoq-toshkentov**
image

### CODE REVIEW

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: packages/ui/src/info-tooltip/InfoTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        children,
+        iconButtonProps,
+        fontSize = 'small',
+        TextProps,
+        title,
+        ...props
+      },
+      ref,
+    ) => {
+      const [isTooltipOpen, setIsTooltipOpen] = useState(false);
```
[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx:44]: no need I will remove it and just paste some props for that

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx:44]: what about if I use useImparativeHandle for to connect with its ref?

[the remark this answers is not in the document]
Comment (doniyor2109) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx:44]: @umrzoq-toshkentov Hmm, why do we need to connect with its ref?

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx]: yes we will connect with its ref e.g

```
const ParentComponent = () => {
   const ref = useRef<RefProps>(null)
   
   const onOpen = () => {
      //if you want can pass props and also can connect with two data binding 
      ref.current.openTooltip()
   }
   
   return (
     <Infotooltip ref={ref} />
   )
}
```

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx]: first of all it avoids from re-render and gives access to create own reusable component life cycle which is performant. We can easily connect child component with parent component and can utilize two-way data binding.

[the remark this answers is not in the document]
Comment (doniyor2109) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx]: @umrzoq-toshkentov I think it is easier to control by props

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx]: okay I will change to props

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/info-tooltip/InfoTooltip.tsx]: updated to props

**Review by doniyor2109 — APPROVED** [review]

**Review by normurod72 — APPROVED** [review]

**Pull request merged**